src/network/changesManager/changesmanager.h

```diff
@@ -0,0 +1,29 @@
+#ifndef CHANGESMANAGER_H
+#define CHANGESMANAGER_H
+
+#include <QString>
+#include <iterator>
+#include <algorithm>
+#include "networkbase.h"
```

Comment:
it's better to declare
(
 "change.h"
 "networkbase.h"
)
above
(
#include QString
#include iterator
#include algorithm
)

Reply:
This is outdated file and is absent in context of build